recipes/llspy-slm/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "llspy-slm" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f279eec9c2920378ea0eb8ca097d62f9ed2e5231fe585c3b2cd5a9581d9b7099
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - slmgen = slmgen.slmwindow:main
+  script_env:
+    - QT_QPA_PLATFORM=offscreen
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numba
```

Comment:
This is only required for python<3.10: https://github.com/tlambert03/llspy-slm/blob/d959f3fd45468ed1df4c29102f300ed76e532660/setup.cfg#L35

Since you're the maintainer, what is preferred?

Reply:
that was added earlier on when numba didn't have wheels on pypi for 3.10.  on conda, I'd prefer to include it

Reply:
Ahh ok yes of course then let's keep it.